### DISCUSSION TITLE
Strange highlighting

### DISCUSSION THREAD

**Discussion by Marinorama** [gsantner/markor#1822] [General] [answered]
When I enter a place name like "St.Georg" in a Markdown file, it appears (in edit mode) in blue and in smaller font. Why? What kind of syntax is highlighted?

**Comment by gsantner** [comment, marked as answer]
blue small highlighting = URL/domain highlighting. while yes it might be not one in that case, st.video would be one 😄 . Anyway in case of Sankt Georg you would write `St. Georg` not `St.Gerog`

**Comment by harshad1** [comment]
This happens all the time when I am editing a numbered list `2.foo` gets highlighted as a link until I add the space. We should consider revisiting the link regex
